packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/numberBoxParts/mask.tests.js

```diff
@@ -2087,13 +2087,15 @@ QUnit.module('format: caret boundaries', moduleConfig, () => {
 
         this.input.focus();
         this.clock.tick(CARET_TIMEOUT_DURATION);
-        for(let i = 0; i < 2; ++i) {
-            this.keyboard.caret(3);
-            this.input.trigger('dxclick');
-            this.clock.tick(CARET_TIMEOUT_DURATION);
-        }
 
+        this.keyboard.caret(3);
+        this.input.trigger('dxclick');
+        this.clock.tick(CARET_TIMEOUT_DURATION);
+
+        assert.timeout(100);
```

Comment:
We usually use sinon for timers as a more strict way.
Why do we need this magic timeout?